README.md

```diff
@@ -74,12 +74,15 @@ the application is up to date and can run appropriately.
 A good pattern to support is having an optional argument that is a file path.
 This allows you to support running single tests.
 
+Linting (ie rubocop, jshint, pmd, etc) can also be considered a form of testing. These tend to run faster than tests, so put them towards the beginning of a `script/test` so it fails faster if there's a linting problem.
```

Comment:
`etc` -> `etc.` Also might be nice to monospace the command names (but feel free to ignore that). Sorry for the pedantry!